### DISCUSSION TITLE
Possibility to create a recurent event on the same day at different time.

### DISCUSSION THREAD

**Discussion by philippe-leeroy** [solspace/craft-calendar#237] [Feature Requests]
Hello, I might be missing something, but is it possible to create an event at different times on the same day...

For example, a show is planned for January 5 at 7 and 9 p.m.

Thanks in advance.

**Comment by kjmartens** [comment]
Unfortunately this is currently not possible. You will need to add a copy of that event to have that level of control. 🙂

**Comment by philippe-leeroy** [comment]
It's a feature request, I know that it's not in place...

**Reply by kjmartens** [reply]
For sure! Thanks for your feedback 🙂

**Comment by jowhannez** [comment]
+1 I have a client that is very interested in this feature, any plans of implementation?

**Comment by A-Winther** [comment]
This feature is absolutely missing. Any plans of implementation?
